test/test.module.ts

```diff
@@ -0,0 +1,13 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
```

Comment:
not necessary, actually env is set on [here](https://github.com/matthewUnowquare/yourpett-back/blob/4b35261b4c8436f929defa9cfdf2a2cd39a2f3ce/src/app.module.ts#L16)